méli-mélo/2021-05-conjunction/and-or.css

```diff
@@ -1,263 +1,286 @@
+  border-width: 3px 0px 3px 0px;
 }
-[class*=cnjnctn-type-]>.cnjnctn-col-90 {
-    flex-basis: 90%
+html:lang(en) .cnjnctn-type-and > [class*=cnjnctn-col]:not(:first-child):before {
+  content: "and";
 }
-[class*=cnjnctn-type-]>.cnjnctn-col-80 {
-    flex-basis: 80%
+html:lang(fr) .cnjnctn-type-and > [class*=cnjnctn-col]:not(:first-child):before {
+  content: "et";
 }
-[class*=cnjnctn-type-]>.cnjnctn-col-75 {
-    flex-basis: 75%
+html:lang(en) .cnjnctn-type-or > [class*=cnjnctn-col]:not(:first-child):before {
+  content: "or";
 }
-[class*=cnjnctn-type-]>.cnjnctn-col-70 {
-    flex-basis: 70%
+html:lang(fr) .cnjnctn-type-or > [class*=cnjnctn-col]:not(:first-child):before {
+  content: "ou";
 }
-[class*=cnjnctn-type-]>.cnjnctn-col-60 {
-    flex-basis: 60%
+[class*=cnjnctn-type-] > .cnjnctn-col-90 {
+  flex-basis: 90%
 }
-[class*=cnjnctn-type-]>.cnjnctn-col-50 {
-    flex-basis: 50%
+[class*=cnjnctn-type-] > .cnjnctn-col-80 {
+  flex-basis: 80%
 }
-[class*=cnjnctn-type-]>.cnjnctn-col-40 {
-    flex-basis: 40%
+[class*=cnjnctn-type-] > .cnjnctn-col-75 {
+  flex-basis: 75%
 }
-[class*=cnjnctn-type-]>.cnjnctn-col-30 {
-    flex-basis: 30%
+[class*=cnjnctn-type-] > .cnjnctn-col-70 {
+  flex-basis: 70%
 }
-[class*=cnjnctn-type-]>.cnjnctn-col-25 {
-    flex-basis: 25%
+[class*=cnjnctn-type-] > .cnjnctn-col-60 {
+  flex-basis: 60%
 }
-[class*=cnjnctn-type-]>.cnjnctn-col-20 {
-    flex-basis: 20%
+[class*=cnjnctn-type-] > .cnjnctn-col-50 {
+  flex-basis: 50%
+}
+[class*=cnjnctn-type-] > .cnjnctn-col-40 {
+  flex-basis: 40%
+}
+[class*=cnjnctn-type-] > .cnjnctn-col-30 {
+  flex-basis: 30%
+}
+[class*=cnjnctn-type-] > .cnjnctn-col-25 {
+  flex-basis: 25%
+}
+[class*=cnjnctn-type-] > .cnjnctn-col-20 {
+  flex-basis: 20%
 }
 [class*=cnjnctn-type-].cnjnctn-xs {
-    flex-direction: row;
+  flex-direction: row;
 }
-[class*=cnjnctn-type-].cnjnctn-xs:not(.brdr-0)>[class*=cnjnctn-col] {
-    padding-left: 0;
-    padding-right: 0;
+[class*=cnjnctn-type-].cnjnctn-xs:not(.brdr-0) > [class*=cnjnctn-col] {
+	min-height: 3em;
+	padding-left: 0;
+	padding-right: 0;
 }
-[class*=cnjnctn-type-].cnjnctn-xs>[class*=cnjnctn-col]:not(:first-child):after {
-    background-image: linear-gradient(to bottom, #6f6f6f 0.3em, rgba(0,0,0,0) 0.3em, rgba(0,0,0,0) 2.35em, #6f6f6f 2.35em);
+[class*=cnjnctn-type-].cnjnctn-xs > [class*=cnjnctn-col]:not(:first-child):after {
+  border-image: linear-gradient(to bottom, #6f6f6f 0.3em, #6f6f6f 0.3em, transparent 0.3em, transparent 2.35em, #6f6f6f 2.35em, #6f6f6f 2.35em) 1 100%;
+  border-left: 3px solid #6f6f6f;
 }
-[class*=cnjnctn-type-].cnjnctn-xs>[class*=cnjnctn-col]:not(:first-child) {
-    margin-left: 1.4em;
-    position: relative;
+@media (prefers-contrast: more) {
+  [class*=cnjnctn-type-].cnjnctn-xs > [class*=cnjnctn-col]:not(:first-child):after {
+    border-image: linear-gradient(to bottom, #ffffff 0.3em, #ffffff 0.3em, transparent 0.3em, transparent 2.35em, #ffffff 2.35em, #ffffff 2.35em) 1 100%;
+  }
```

Comment:
I tested that CSS `prefers-contrast: more` and my testing showed it acceptable in our currently supported browser list.

Here a summary of my test result:

* Windows Chrome: work as expected
* Windows Edge: work as expected
* Windows Firefox dev: It is broken, the user can read the conjunction word "and" and "or" because there is a vertical black bar in the middle
* MacOS Big Sur Chrome: Don't work as expected, but the text is readable, the white border gradient only affect the circle around the word, not the word itself.
* MacOS Big Sur Safari: Don't work as expected, but the text is readable, the white border gradient only affect the circle around the word, not the word itself.

I am going to attache those screen capture as a comment in the PR, as this inline comment are only informative